influunt-app/app/views/directives/influunt-revisao/_tabelas-entre-verdes.html

```diff
@@ -0,0 +1,50 @@
+<div class="col-lg-12 col-sm-12 no-padding padding-top-20 ibox-content">
+  <label for="gsTabelasEntreVerdes">{{ 'controladores.configuracao_grupo' | translate }}</label>
+  <select class="form-control m-b" id="gsTabelasEntreVerdes"
+      data-ng-model="gsTabelasEntreVerdes"
+      data-ng-init="gsTabelasEntreVerdes = currentGruposSemaforicos[0]"
+      data-ng-options="gs as 'G'+gs.posicao for gs in currentGruposSemaforicos | orderBy: 'posicao'">
+  </select>
+
+  <label for="gsTabelasEntreVerdes">{{ 'controladores.entreVerdes.titulo' | translate }}</label>
+  <select class="form-control m-b" id="gsTabelasEntreVerdes"
+      data-ng-model="tabelaEntreVerdeSelecionada"
+      data-ng-options="tev as tev.descricao for tev in dadosCurrentTabelaEntreVerdes | filter: {grupoSemaforicoIdJson: gsTabelasEntreVerdes.idJson}"></select>
+</div>
+
+<div class="col-sm-12 no-padding padding-top-20">
+  <div class="col-sm-12 no-padding-left">
+    <h3>
+      <strong>{{ tabelaEntreVerdeSelecionada.descricao }}</strong>
+    </h3>
+  </div>
+  <div class="col-sm-3 no-padding-left"></div>
+  <div class="col-sm-3 no-padding-left">
+    <p>{{ 'controladores.revisao.amarelo' | translate }}</p>
```

Comment:
Essa tabela é para Grupo semaforico veicular e pedestre, se for tem que alterar o nome pois `Amarelo`é para grupo veicular temos que colocar `Amarelo/Vermelho Intermitente`